mantidproject/mantid
# Engggui: include default Engin-X full calibration in settings

Fixes #14590.

To test:
- Open the EnggDiffraction custom interface
- Check that the factory default for "Pixel calibration" in the "Settings" tab is the new fileCheck 
- Go to "Calibration" tab and run a calibration with standard run numbers (previous PRs) and see that it runs without issues
- Check that the documentation updates are correct

This calibration is not perfect yet but seems good enough for a first usable version.

If you have previous settings, you can clean the value of "pixel calibration" by removing `pixel-calib-filename` from your system QSettings which can be found:
- on windows: open the regedit, and find/look for `EnggDiffractionView`. It should be found in `Mantid/CustomInterfaces/EnggDiffractionView` (usually inside `HKEY_CLASSES_ROOT`, `HKEY_CURRENT_USER`).
- on linux, the line `EnggDiffractionView\pixel-calib-filename=` from `~/.config/Mantid/MantidPlot.conf`
